source/usage-examples/delete-operations.txt

```diff
@@ -2,7 +2,8 @@
 Delete Operations
 =================
 
-.. default-domain:: mongodb
+.. meta::
+   :description: See examples of deleting data from MongoDB by using the {+driver-long+}.
```

Comment:
Comment; the phrase "deleting data from" sounds awkward. could it still be "deleting data IN.."?

Reply:
Hm, I think I've always heard the preposition "from" associated with "delete".
E.g. "Delete from your computer" vs "Delete in your computer"
Have you heard the latter before?

Reply:
The way i was thinking about it was that we are working "in" MongoDB and any operations are executed from that point of view. But I see your point too!